### PR TITLE
Fixing ci issues after docker merge

### LINKED ISSUE #316: CI issues
As a consequence of recent merge, CI is failing on all pull requests.
Root causes to fix:

- docker-compose in Travis uses a different pathway (not through the new docker procedure) and it requires different configuration. As a temp WA, will support both configurations. In the future will open an enhancement to have CI usethe same method as dev environment, so as to cover spin up failures

- vault tests are failing due to changes in vault container made to support docker pathway

-Flake8: all code needs to be converted to proper standards

### PR DESCRIPTION
**Pull request by leonicus**
Allow home directory to be passed as env to docker-compose, so it can
support both pathways

Fix PEP8 issues

Copy relevant files to vault container

Fixes #316

### CODE REVIEW

**Comment by leonicus** [comment]
@DanielLitvak changed:)

File: requirements-docker.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ Click==7.0
 pyOpenSSL==19.1.0
 python-dotenv==0.10.3
 python-vagrant==0.5.15
+j2cli
```
Comment (lioramilbaum) [inline requirements-docker.txt]: package version is not locked

Comment (leonicus) [reply requirements-docker.txt]: Fixed

**Comment by leonicus** [comment]
> All of the changes are related to CI?

Yep, there were several failures in CI, all ofthem addressed here and nothing more :)

File: requirements-docker.txt [hunk 2]
```diff
@@ -3,3 +3,8 @@ Click==7.0
 pyOpenSSL==19.1.0
 python-dotenv==0.10.3
 python-vagrant==0.5.15
+<<<<<<< HEAD
```
Comment (lioramilbaum) [inline requirements-docker.txt]: Have you tested this code?

Comment (leonicus) [reply requirements-docker.txt]: Sorry, git issues again :( Have to get rid of the habit of pulling after rebasing....its fixed now. And yes, the code is teste (without the conflict lines of course)

**Pull request merged**